frontend/src/components/Calendar/calendar.css

```diff
@@ -0,0 +1,183 @@
+* {
+    font-family: 'Montserrat', sans-serif;
+    font-style: normal;
+}
+
+.calendar {
+    width: 900px;
+    height: 600px;
+    display: flex;
+    flex-direction: column;
+    margin-top: 2rem;
+    margin-left: auto;
+    margin-right: auto;
+    background-color: #ffffff;
+
+}
+
+.calendar-header {
+    width: 100%;
+    height: 100px;
+    margin: auto;
+    display: flex;
+    align-items: left;
+    justify-content: space-between;
+    font-style: bold;
+
+
+}
+
+.title {
+    width: 25%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+}
+
+.title h2 {
+    margin: auto;
+}
+
+.tools {
+    width: 23%;
+    height: 150%;
+    display: flex;
+    align-items: center;
+}
+
+.tools button {
+    float: left;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    background-color: #ffffff;
+    border: .5px solid #d9d9d9;
+}
+
+.tools button span:hover {
+    color: #99cccc;
+}
+
+.today-button {
+    background-color: white;
+    color: black;
+    border: .5px solid #e9e9e9;
+    text-align: center;
+    padding: 10px 20px;
+    border-radius: 5px;
+    cursor: pointer;
+    font-size: 10px;
+    margin: 4x 2px;
+    font-size: bold 10px;
+    font-weight: bold;
+}
+
+.today-button:hover {
+    background-color: #EDEDED;
+
+}
+
+.prev-button {
+    background-color: white;
+    color: black;
+    border: .5px solid #e9e9e9;
+    text-align: center;
+    padding: 10px 20px;
+    border-radius: 5px;
+    cursor: pointer;
+    font-size: 10px;
+    margin: 4x 2px;
+    font-size: bold 10px;
+    font-weight: bold;
+}
+
+.prev-button:hover {
+    background-color: #EDEDED;
+
+}
+
+.next-button {
+    background-color: white;
+    color: black;
+    border: .5px solid #e9e9e9;
+    text-align: center;
+    padding: 10px 20px;
+    border-radius: 5px;
+    cursor: pointer;
+    font-size: 10px;
+    margin: 4x 2px;
+    font-size: bold 10px;
+    font-weight: bold;
+}
+
+.next-button:hover {
+    background-color: #EDEDED;
+
+}
+
```

Comment:
lines

Reply:
gonna stop making comments on them going forward, keep an eye out for them please